src/backend/utils/cache/ag_cache.c

```diff
@@ -633,7 +633,7 @@ static void flush_label_oid_cache(void)
 {
     HASH_SEQ_STATUS hash_seq;
 
-    hash_seq_init(&hash_seq, label_name_graph_cache_hash);
+    hash_seq_init(&hash_seq, label_oid_cache_hash);
```

Comment:
What was your reasoning for doing this? Our goal is to move away from oids?

Reply:
We doing tests with dump & restore of the database with AGE extension and some graph data. After restore we have broken oids in the ag_catalog.* tables. Which can be easily fixed and I will commit the fix procedure later. But even after the oid fix we still have errors in the `flush_label_oid_cache(void)`. I think that using `label_name_graph_cache_hash` is just a typo. At least after my fix we see no errors with broken cache anymore after dump & restore and oid fixes

Reply:
Eliminating oids would make dump&restore a breeze, but probably will impact query speed ? I did not spent much time in AGE internals to participate in decision making, I guess.